tools/jest.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
   roots: ['<rootDir>/tests'],
   testPathIgnorePatterns: ['<rootDir>/src/infrastructure/plugins/'],
-  coveragePathIgnorePatterns: ['<rootDir>/src/infrastructure/plugins/'],
+  coveragePathIgnorePatterns: ['<rootDir>/src/infrastructure/plugins/', '<rootDir>/src/interfaces/'],
```

Comment:
This is just FYI, in the next PR we are going to remove both ignore patterns `testPathIgnorePatterns` and `coveragePathIgnorePatterns`.